FPSCamera/FPSCamera.cs

```diff
@@ -32,6 +34,18 @@ public static void Initialize(LoadMode mode)
                 instance.citizenCamera = instance.gameObject.AddComponent<CitizenCamera>();
                 instance.citizenCamera.vehicleCamera = instance.vehicleCamera;
                 editorMode = false;
+
+                foreach(System.Reflection.Assembly assembly in System.AppDomain.CurrentDomain.GetAssemblies())
+                {
+                    foreach(System.Type type in assembly.GetTypes())
```

Comment:
Seems like this might be causing a `ReflectionTypeLoadException` when the IPT2 assembly isn't available:

```
The Mod C:\Program Files (x86)\Steam\steamapps\workshop\content\255710\650805785 [FPSCamera.dll] has caused an error [ModException]

Details:
System.Reflection.ReflectionTypeLoadException: The classes in the module cannot be loaded.
  at (wrapper managed-to-native) System.Reflection.Assembly:GetTypes (bool)
  at System.Reflection.Assembly.GetTypes () [0x00000] in <filename unknown>:0 
  at FPSCamera.FPSCamera.Initialize (LoadMode mode) [0x00000] in <filename unknown>:0 
  at FPSCamera.ModLoad.OnLevelLoaded (LoadMode mode) [0x00000] in <filename unknown>:0 
  at LoadingWrapper.OnLevelLoaded (UpdateMode mode) [0x00000] in <filename unknown>:0 
```

Reply:
IPT2 is not what's causing it. I had it enabled but still got the error. It has something to do with the ```assembly.GetTypes()```. https://docs.microsoft.com/en-us/dotnet/api/system.reflection.assembly.gettypes?view=netframework-4.8

Reply:
I found this: https://haacked.com/archive/2012/07/23/get-all-types-in-an-assembly.aspx/

It makes the exception go away but it finds IPT2 even when it's not enabled.  I added a try/catch in the loop to catch the error and turn ipt mode off if IPT is not enabled. I hope it only throws that error if IPT is disabled but otherwise it's not a very big deal.

Reply:
I hope it works now..

Reply:
Perhaps it might be better to grab it via the in-game method (see [IPT2](https://github.com/bloodypenguin/ImprovedPublicTransport/blob/f8f037b9709e9c6f331f31397f03cb4f580a9cf7/Utils.cs) and [TM:PE](https://github.com/krzychu124/Cities-Skylines-Traffic-Manager-President-Edition/blob/95d7807c6cbb36f919629a777dd34c4fdb3bdc7b/TLM/TLM/Util/ModsCompatibilityChecker.cs)).

Might be easier for me to merge this in as-is and apply the fix afterward, if you want.

Reply:
You decide how you want to do it. I like the inputs you are giving me, they are very helpful. ```PluginManager``` sounds like a good idea to use. I can try to implement it now, otherwise I think I'll have it done tomorrow.

Reply:
I copied some code from TM:PE so now it checks with ```PluginManager```. The only catch is that it only recognizes IPT2 from the workshop. Currently it uses ```info.publishedFileID``` to get the mod info. I tried using ```info.name``` but that only gave me the name of the folders containing the mod. I don't know a better way to do it but I'm sure there is one..